src/extension/background-script/connectors/lnd.ts

```diff
@@ -63,6 +65,17 @@ class Lnd implements Connector {
     });
   }
 
+  async checkPayment(args: CheckPaymentArgs): Promise<CheckPaymentResponse> {
```

Comment:
to be more generally used ideally we return more invoice data and maybe even call the function `getInvoice`? 
sadly it seems lndhub/lnbits to not really return much useful invoice data.